emails/message.py

```diff
@@ -260,6 +260,9 @@ def _build_root_message(self, message_cls=None, **kw):
         if self.cc:
             self.set_header(msg, 'Cc', ", ".join([self.encode_address_header(addr) for addr in self.cc]), encode=False)
 
+        if self.bcc:
```

Comment:
I thought Bcc should be used in SMTP MAIL TO command only.
Is there any reason to include it in message headers?